### PR TITLE
Add Journal of Educational Data Mining

### PR DESCRIPTION
**Pull request by jooyoungseo**
# Changes

- Added Journal of Educational Data Mining LaTeX template according to the code of conduct.
- Added a `jedm_article()` function in `article.R`.
- Added related info to `README.md`; `DESCRIPTION`; and `NEWS.md`.

# Building

- Used `Rd2roxygen::rab()` function to build `rticles` package after change has been made.

# Testing

I have confirmed that `rticles::jedm_article()` output format works.

# Signed-Form

- I have signed and submitted the RStudio form to J.J.

### CODE REVIEW

File: inst/rmarkdown/templates/jedm_article/skeleton/skeleton.Rmd [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  \end{tabular}
+\end{table}
+
+# Appendices
+
+Supplementary technical material (e.g., mathematical proofs or descriptions of experimental procedures) should be collected in an appendix at the end of the paper (before the acknowledgements and the references sections).
+
+# Footnotes and acknowledgments
+
+Footnotes should be used sparingly and indicated by consecutive superscript numbers in the text. Material to be footnoted should appear at the bottom of the page on which it is referenced. Acknowledgments and grant numbers should be put into a separate `Acknowledgment' section right before the list of references.
```
Comment (cderv) [inline inst/rmarkdown/templates/jedm_article/skeleton/skeleton.Rmd:81]: I think that is the wrong quote after Acknowledgment ? Am I right ?

Comment (yihui) [reply inst/rmarkdown/templates/jedm_article/skeleton/skeleton.Rmd:81]: FYI, `` ` '`` is the LaTeX syntax for a pair of curly single quotes (similarly, ``` `` ''``` is curly double quotes). However, here we shouldn't use the LaTeX syntax, but just two straight single quotes `` ' '``.

Comment (cderv) [reply inst/rmarkdown/templates/jedm_article/skeleton/skeleton.Rmd:81]: Oh thank you ! I did not know that.

**Comment by cderv** [comment]
@Yihui, seen with @jooyoungseo , this will not be ready for next release so I added a new milestone to the next version to follow this

**Comment by yihui** [comment]
@cderv Sounds good. I no longer use milestones to manage releases because I found it hard to actually stick to the milestone and often ended up with something like this:

image

If we are going to use milestones at all, personally I like Pandoc's single milestone, i.e. "next release": https://github.com/jgm/pandoc/milestones

I feel rticles is too small a project to warrant using milestones to manage :) That said, I'll just leave it to you to decide how you want to manage this project. I'm okay with any method.

**Comment by cderv** [comment]
Oh yeah next release is surely enough. 

The need here is a way to track that this will not be dealt with for next release. It think it is important to track what we want to cover for next release, unless we use another project management tool than Github included ones.

So yes one "next release" milestone is fine for that, or maybe just a label would do it. But tracking is different.

**Review by cderv — APPROVED** [review]
This was long overdue. I made the necessary change for this one. 
I believe this is working as expected now.

**Comment by yihui** [comment]
Perfect. Thanks!

**Pull request merged**